src/indexer/merger.rs

```diff
@@ -176,16 +177,39 @@ impl IndexMerger {
         schema: Schema,
         index_settings: IndexSettings,
         segments: &[Segment],
+    ) -> crate::Result<IndexMerger> {
+        let delete_bitsets = segments.iter().map(|_| None).collect_vec();
+        Self::open_with_custom_alive_set(schema, index_settings, segments, delete_bitsets)
+    }
+
+    // Create merge with a custom delete set.
+    // For every Segment, a delete bitset can be provided, which
+    // will be merged with the existing bit set. Make sure the index
+    // corresponds to the segment index.
```

Comment:
Please document what is the semantics for None because it can go different ways.